bitbots_vision_tools/scripts/colorspace_tool.py

```diff
@@ -290,7 +290,7 @@ def save(self, filename):
             blue = blue
         )
 
-        filename = '{}_{}.txt'.format(filename, output_type)
+        filename = f'{filename}_{output_type}.pickle'
```

Comment:
This syntax is cool, but only supported in python3

Reply:
Well, the shebang says py3, but i can change this